rialto_airflow/dags/harvest.py

```diff
@@ -83,92 +84,87 @@ def load_authors(snapshot):
         Load the authors data from the authors CSV into the database.
         """
         authors.load_authors_table(snapshot)
-        return snapshot
 
     @task()
     def dimensions_harvest(snapshot):
         """
         Fetch the data by ORCID from Dimensions.
         """
-        jsonl_file = dimensions.harvest(snapshot, limit=harvest_limit)
-
-        return jsonl_file
+        dimensions.harvest(snapshot, limit=harvest_limit)
 
     @task()
     def openalex_harvest(snapshot):
         """
         Fetch the data by ORCID from OpenAlex.
         """
-        jsonl_file = openalex.harvest(snapshot, limit=harvest_limit)
-
-        return jsonl_file
+        openalex.harvest(snapshot, limit=harvest_limit)
 
     @task()
     def wos_harvest(snapshot):
         """
         Fetch the data by ORCID from Web of Science.
         """
-        jsonl_file = wos.harvest(snapshot, limit=harvest_limit)
-
-        return jsonl_file
+        wos.harvest(snapshot, limit=harvest_limit)
 
     @task()
-    def sul_pub_harvest(snapshot):
+    def sulpub_harvest(snapshot):
         """
         Harvest data from SUL-Pub.
         """
-        jsonl_file = sul_pub.harvest(
-            snapshot, sul_pub_host, sul_pub_key, limit=harvest_limit
-        )
+        sul_pub.harvest(snapshot, sul_pub_host, sul_pub_key, limit=harvest_limit)
 
-        return jsonl_file
+    @task_group()
+    def harvest_pubs(snapshot):
+        dimensions_harvest(snapshot)
+        openalex_harvest(snapshot)
+        wos_harvest(snapshot)
+        sulpub_harvest(snapshot)
 
     @task()
```

Comment:
We don't actually need all the jsonl files. We were previously using them to express dependencies between tasks, but that is now no longer needed since we can use the task group as a workflow dependency.